scripts/eagle_renumber.py

```diff
@@ -78,14 +78,14 @@ def resolve_unit(value, unit_prefix):
       '<part name="{}(.+?)".*value="([\.0-9]+)([{}]?)(.*?)".*[/]?>'.\
             format(prefix, ''.join(known_unit_prefixes.keys()))
             )
-sch_instance_tag_re = re.compile('<instance part="{}(.+?)".*'.format(prefix))
-sch_pinref_tag_re = re.compile('<pinref part="{}(.+?)".*'.format(prefix))
+sch_instance_tag_re = re.compile('<instance part="{}([0-9]+?)".*'.format(prefix))
```

Comment:
You should make this `[\.0-9]`, like the match above from the part tag. You want to allow periods as well for things like `1.5k`